test-framework/sudo-compliance-tests/src/sudoedit/kill-sudoedit.sh

```diff
@@ -0,0 +1,18 @@
+until [ -f /tmp/barrier ]; do 
+    sleep 0.1
+done
+
+target="$1"
+shift
+
+# when sudoedit runs there are two sudo processes as sudoedit spawns
+# a child process. We assume the parent is the smallest of the returned PIDs.
```

Comment:
This is not true in general, but should be true within the pid namespace of the container as we start less than 32k processes inside each container. And we shouldn't care about people setting an absurdly small `/proc/sys/kernel/pid_max` value.

Reply:
In any case this was copied from the existing test scripts.